### DISCUSSION TITLE
[FR] Support Gravatar profile images by default

### DISCUSSION THREAD

**Discussion by bertoost** [craftcms/cms#8820] [Ideas]
### Description

It would be nice to support Gravatar profile images by default for users in de system.
Or a way that a plugin can get involved. Which is maybe even better, if someone has a different source for profile images.

### Additional info

- Craft version: 3.4.15
- PHP version: 7.4.4

**Comment by joshuabaker** [comment]
A hook for this would be great! Gravatar support out of the box would be amazing though.
